dashboard-plugin/client-version.js

```diff
@@ -5,16 +5,18 @@ module.exports = ({
   fallbackEntryURL,
   fallbackRemoteVar,
   dashboardTimeout,
+  enableCacheBuster,
 }) => {
-  fallbackRemoteVar = fallbackRemoteVar || remoteName;
-  fallbackEntryURL = fallbackEntryURL || '';
+  fallbackRemoteVar = fallbackRemoteVar || remoteName
+  fallbackEntryURL = fallbackEntryURL || ''
+  enableCacheBuster = enableCacheBuster || false
```

Comment:
re-add semicolons. switch to fallbackEnableCacheBuser so it's clear the cachebuster is for fallback only